### ISSUE TITLE
Path to image files is absolute

### ISSUE THREAD

**Issue by mohelten** [notaapp/issues#123]
I would like to have relative paths to the md file if I add images.

I tried this preference:

![image](https://user-images.githubusercontent.com/1385665/138909962-d7cccaa9-a524-4300-abb9-5c5248205ac9.png)

The images are added in the same folder. That is correct.
But if I view the md file the absolute path to the file is used.
`![](/absolutepath/image.jpg)`

I would like to have relative paths here so I can easily move md and asset files to another place.

If I manually change the markdown to:
`![](./image.jpg)`

it works also in the Nota preview.

So maybe there is a trick or a minor change in the preferences that will result in my requirement.

**Comment by evaldsurtans** [comment]
Yes this is very annoying, please try to open files in Typora and compare - relative paths work much better across other MD editors. Please fix it.

**Comment by pcatterall7** [comment]
+1 for this since the other markdown editors I use (1writer, IAWriter) don't play nice with the absolute links

**Comment by astoilkov** [comment]
I agree. We will do something about this.

**Comment by reorx** [comment]
This is also a must have for me, I organize my writings in a way that attachments are placed in the `images` folder relative to the markdown file. This is a recommended way by Hugo, so that `images` folder could be recognized as [Page Resources](https://gohugo.io/content-management/page-resources/) of the [bundle](https://gohugo.io/content-management/page-bundles/).

I think Obsidian's approach is very good, you can take it as an inspiration.
![image](https://user-images.githubusercontent.com/405972/187145352-ba2b41d0-591a-4caf-9da3-4d3a1bb798ca.png)

(But I think Obsidian's writing experience is far worse than it's functionality, this is where Nota wins :)

**Comment by decisivedevops** [comment]
had this issue as well! I added few images to my github README file, all the images go into `_docs` folder from nota, and in nota, all the images are visible.

BUT in github, due to only filenames (not path), the images are not shown.

IN NOTA:

![CleanShot 2023-02-14 at 10 50 53](https://user-images.githubusercontent.com/95153836/218646510-1ffbe9d3-ce75-4ece-a4fe-bd3f2b907112.jpg)

IN GITHUB:
![CleanShot 2023-02-14 at 10 52 07](https://user-images.githubusercontent.com/95153836/218646638-90378f15-f0b7-4b44-aa1c-db7f3cae2a53.jpg)

Comments above have suggested few tools that solve this.

waiting for this feature to be added.

**Comment by reorx** [comment]
BTW, haven't seen any updates for a while. Is this project still ongoing?

**Comment by astoilkov** [comment]
We've released 4 versions in the last month. 2 minor updates and 2 patches. This has gone under the radar because we still haven't enabled automatic updates to these versions because they introduce big changes in the codebase and we are upgrading users gradually.

**Comment by newsletters001** [comment]
This is also a showstopper for me as I need compatibility with other MD editors that I would like to use.
Are there any news on this issue?
